Library/Formula/syntaxerl.rb

```diff
@@ -0,0 +1,26 @@
+class Syntaxerl < Formula
+  homepage "https://github.com/ten0s/syntaxerl"
+  url "https://github.com/ten0s/syntaxerl/archive/0.8.0.tar.gz"
+  sha256 "4c5a6390a7d28869bf44cc831ae59cb69480a8481853de8dd59672b7cf5bff7b"
+
+  depends_on "erlang"
+
+  def install
+    system "make"
+    bin.install "syntaxerl"
+  end
+
+  test do
+    require "open3"
+    (testpath/"app.config").write "[{app,[{arg1,1},{arg2,2}]}]."
+    Open3.popen3("#{bin}/syntaxerl", testpath/"app.config") do |stdin, stdout, _|
```

Comment:
You could use `shell_output("#{bin}/syntaxerl #{testpath}/app.config")` to get the output of the command and then test it, e.g.: `assert_equal "", shell_output("#{bin}/syntaxerl #{testpath}/app.config")`.